chain/validation/keymanager.go

```diff
@@ -87,7 +86,8 @@ func (k *KeyManager) newBLSKey() *wallet.Key {
 	// FIXME: bls needs deterministic key generation
 	//sk := ffi.PrivateKeyGenerate(s.blsSeed)
 	// s.blsSeed++
-	sk := ffi.PrivateKeyGenerate()
+	sk := [32]byte{}
+	sk[0] = uint8(k.blsSeed+1) // hack to keep gas values determinist
```

Comment:
:fire:

Reply:
Waiting on this: https://github.com/filecoin-project/filecoin-ffi/issues/38#issuecomment-594900318